### ISSUE TITLE
test error 

### ISSUE THREAD

**Issue by huihuixiaohuihui** [open-mmlab/mmocr#433]
hello, I tested the picture and found that the prediction effect of some pictures is the result of the picture rotating 180 degrees. I don't modified the config file. pls help me, thanks.
my config information.

_base_ = ['../../_base_/runtime_10e.py']

# optimizer
optimizer = dict(type='Adam', lr=1e-4)
optimizer_config = dict(grad_clip=None)
# learning policy
lr_config = dict(policy='step', step=[41, 80])
total_epochs = 600

model = dict(
    type='PSENet',
    backbone=dict(
        type='ResNet',
        depth=50,
        num_stages=4,
        out_indices=(0, 1, 2, 3),
        frozen_stages=-1,
        norm_cfg=dict(type='SyncBN', requires_grad=True),
        init_cfg=dict(type='Pretrained', checkpoint='torchvision://resnet50'),
        norm_eval=True,
        style='caffe'),
    neck=dict(
        type='FPNF',
        in_channels=[256, 512, 1024, 2048],
        out_channels=256,
        fusion_type='concat'),
    bbox_head=dict(
        type='PSEHead',
        text_repr_type='quad',
        in_channels=[256],
        out_channels=7,
        loss=dict(type='PSELoss')),
    train_cfg=None,
    test_cfg=None)

dataset_type = 'IcdarDataset'
data_root = 'datasets_ori'
img_norm_cfg = dict(
    mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375], to_rgb=True)

train_pipeline = [
    dict(type='LoadImageFromFile', color_type='color_ignore_orientation'),
    dict(
        type='LoadTextAnnotations',
        with_bbox=True,
        with_mask=True,
        poly2mask=False),
    dict(type='ColorJitter', brightness=32.0 / 255, saturation=0.5),
    dict(type='Normalize', **img_norm_cfg),
    dict(
        type='ScaleAspectJitter',
        img_scale=[(3000, 736)],  # unused
        ratio_range=(0.5, 3),
        aspect_ratio_range=(1, 1),
        multiscale_mode='value',
        long_size_bound=1280,
        short_size_bound=640,
        resize_type='long_short_bound',
        keep_ratio=False),
    dict(type='PSENetTargets'),
    dict(type='RandomFlip', flip_ratio=0.5, direction='horizontal'),
    dict(type='RandomRotateTextDet'),
    dict(
        type='RandomCropInstances',
        target_size=(640, 640),
        instance_key='gt_kernels'),
    dict(type='Pad', size_divisor=32),
    dict(
        type='CustomFormatBundle',
        keys=['gt_kernels', 'gt_mask'],
        visualize=dict(flag=False, boundary_key='gt_kernels')),
    dict(type='Collect', keys=['img', 'gt_kernels', 'gt_mask'])
]
test_pipeline = [
    dict(type='LoadImageFromFile', color_type='color_ignore_orientation'),
    dict(
        type='MultiScaleFlipAug',
        img_scale=(2240, 2200),
        flip=False,
        transforms=[
            dict(type='Resize', img_scale=(2240, 2200), keep_ratio=True),
            dict(type='Normalize', **img_norm_cfg),
            dict(type='Pad', size_divisor=32),
            dict(type='ImageToTensor', keys=['img']),
            dict(type='Collect', keys=['img']),
        ])
]

data = dict(
    samples_per_gpu=8,
    workers_per_gpu=2,
    val_dataloader=dict(samples_per_gpu=1),
    test_dataloader=dict(samples_per_gpu=1),
    train=dict(
        type=dataset_type,
        ann_file=data_root + '/instances_train.json',
        img_prefix=data_root + '/imgs',
        pipeline=train_pipeline),
    val=dict(
        type=dataset_type,
        ann_file=data_root + '/instances_test.json',
        img_prefix=data_root + '/imgs',
        pipeline=test_pipeline),
    test=dict(
        type=dataset_type,
        ann_file=data_root + '/instances_test.json',
        img_prefix=data_root + '/imgs',
        pipeline=test_pipeline))

evaluation = dict(interval=10, metric='hmean-iou')

my command: python tools/det_test_imgs.py datasets_ori/imgs/test/ datasets_ori/imgs/testimg.txt ./configs/textdet/psenet/psenet_r50_fpnf_600e_icdar2015.py ./pser50pre.pth --out-dir ./pipepretrain --score-thr 0.1
![1401](https://user-images.githubusercontent.com/45276031/129335971-9d8a98e7-89ab-4584-84f9-7e88ba39c589.jpg)

**Comment by huihuixiaohuihui** [comment]
When I'm training, it's hard to reduce the loss value， and I think my data is easier than CTW,ICDAR data.

**Comment by gaotongxiao** [comment]
How has your annotation been made? Consider changing your test_pipeline:
```
dict(type='LoadImageFromFile', color_type='color_ignore_orientation'),
```
to 
```
dict(type='LoadImageFromFile'),
```
For more info, see the notes under the table: https://mmocr.readthedocs.io/en/latest/datasets.html#text-detection and this PR: https://github.com/open-mmlab/mmocr/pull/375

**Comment by huihuixiaohuihui** [comment]
> How has your annotation been made? Consider changing your test_pipeline:
> 
> ```
> dict(type='LoadImageFromFile', color_type='color_ignore_orientation'),
> ```
> 
> to
> 
> ```
> dict(type='LoadImageFromFile'),
> ```
> 
> For more info, see the notes under the table: https://mmocr.readthedocs.io/en/latest/datasets.html#text-detection and this PR: #375
Thanks a lot for replying, I have solve it follow your advise.  Can you tell me the function of this parameter('color_ignore_orientation') by the way, and I want to know how to display polygon.  set text_repr_type = 'poly' ?

**Comment by gaotongxiao** [comment]
> Thanks a lot for replying, I have solve it follow your advise.  Can you tell me the function of this parameter('color_ignore_orientation') 

You can find the explanation in this PR https://github.com/open-mmlab/mmocr/pull/375. 

> by the way, and I want to know how to display polygon.  set text_repr_type = 'poly' ?

Yes.

**Comment by gaotongxiao** [comment]
I'm closing this issue as there is no further update.
